### PR TITLE
Crypto enhancements

### LINKED ISSUE #1: Enhancements to crypto donation impact calculator
1. Incorporate API calls to the Kraken exchange ([API docs](https://www.kraken.com/features/api#get-ticker-info) - no API keys required) to get live price feed of [BTCUSD](https://api.kraken.com/0/public/Ticker?pair=XBTUSD), [ETHUSD](https://api.kraken.com/0/public/Ticker?pair=ETHUSD), and [DAIUSD](https://api.kraken.com/0/public/Ticker?pair=DAIUSD) trading pairs. These links are the API data you'll need to call, but we only care about the price. This value should appear properly inside the Grid ([where value goes in the code](https://github.com/fuguefoundation/ff-impact-calculator/blob/0b3d43034b1d40d8fb4774c82bbc06d6f4d36e6c/src/components/CryptoRichAmI/index.js#L155), make the UX/UI consistent with app) and update as the user changes the amount of their donation.

2. Write a test (in Jest) for this API capability.

3. The user should be able to enter a donation with up to two decimal points (e.g., .01 BTC, 2.25 ETH) and have the calculations in the app still work properly. I believe the problem of input is related to some validation/regex in [`index.js`](https://github.com/fuguefoundation/ff-impact-calculator/blob/0b3d43034b1d40d8fb4774c82bbc06d6f4d36e6c/src/components/CryptoRichAmI/index.js#L64) in either the `parseNumericInput` or `validInteger` functions. I'm not sure if adding decimals subsequently screws up the math, but please ensure that adding decimals still enables the full calculation abilities of the app. 

I have a [working prototype](https://fuguefoundation.org/dev/impact) of the app for reference. See the README.md file in the repo for more information and background.

### PR DESCRIPTION
**Pull request by KiChjang**
Fixes #1.

### CODE REVIEW

**Comment by KiChjang** [comment]
I ended up not refreshing the exchange rate on a set interval, because I didn't think users needed such a feature for this app.

**Comment by fuguefoundation** [comment]
Very cool @KiChjang , I'll take a look soon and get back to you.

File: src/components/CryptoRichAmI/index.js
Comment (fuguefoundation) [inline src/components/CryptoRichAmI/index.js]: Not sure why the onBlur was added (I assume that is what is causing this issue), but you now have to click outside the donation field to validate the form and enable the submit button. You can reproduce this bug on page load if all the fields are empty (accept the household/chidlren, which are pre-populated). If you first choose your crypto and then enter an amount for the donation, you need to click outside the donation field to enable the button. Button should be able to submit once valid donation amount is added.

File: src/components/CryptoRichAmI/index.js
Comment (fuguefoundation) [inline src/components/CryptoRichAmI/index.js]: In addition to showing current market price, this section should show the current price relative to the amount user puts into the donation field. For example if the user selects bticoin and types in 0.5 for the donation, it should show something like Current BTC Price: 10,000 | User amount: 5000

File: src/components/CryptoRichAmI/index.js
Comment (fuguefoundation) [inline src/components/CryptoRichAmI/index.js:282]: Super quick edit here, but you need to repeat what you did here/line 278 inside `const DonationCalculation` as well (i.e., lines 335 and 340). Currently, only half of the calculator is showing and the other half is erroring out, because `exchangeRate` is not included the second time the `calculate` method is called. So just add `exchangeRate` variable again on those two lines and it works.

**Review by fuguefoundation — CHANGES_REQUESTED** [review]
Hey @KiChjang, great work as usual. Just a few things to address before merge. See my comments below. Please take another pass and let me know if you have any questions.

File: src/components/CryptoRichAmI/index.js
Comment (KiChjang) [reply src/components/CryptoRichAmI/index.js]: It's not really an issue but a compromise. We can't immediately parse the numbers on every input, because if you wanted to type "1.23", once you've typed "1.", the parsing algorithm will parse that string and cast that to a 1, which basically removes your decimal point. The only way I can think of fixing this is by hooking into the focus out event and parse the complete string after it's been completely entered.

Comment (KiChjang) [reply src/components/CryptoRichAmI/index.js]: Ok, found another way instead: use a timeout function to hold off parsing the string input for about 700 milliseconds.

**Comment by KiChjang** [comment]
@fuguefoundation I think I've addressed all your comments now, please re-review.

**Comment by fuguefoundation** [comment]
Looks great @KiChjang, will merge and release funds, plus tip for help with the interpolation. Thanks!

**Comment by fuguefoundation** [comment]
https://stackoverflow.com/a/62804878/2649549
@KiChjang thanks, got the answer after posting on SO. Was a routing issue ([docs](https://reactrouter.com/web/api/BrowserRouter)), working now.

**Comment by KiChjang** [comment]
Huh, looks like my initial hunch was correct... it was a routing issue after all!

**Pull request merged**
